sdks/java/io/hadoop-common/src/main/java/org/apache/beam/sdk/io/hadoop/SerializableConfiguration.java

```diff
@@ -94,4 +102,34 @@ public static Configuration newConfiguration(@Nullable SerializableConfiguration
     }
   }
 
+  /**
+   * Loading hadoop configuration from HADOOP_CONF_DIR and YARN_CONF_DIR environment variable
+   *
+   * <p>Currently this makes a shallow copy of the conf directory. If there are cases where a
+   * Hadoop config directory contains subdirectories, this code will have to be fixed.
```

Comment:
Sorry, The comment is wrong. it don't copy of the conf directory but read some xml files, eg: core-site.xml, I will fix it.